_posts/2022-08-23-luz-0-3/luz-0-3.Rmd

```diff
@@ -0,0 +1,206 @@
+---
+title: "luz 0.3.0"
+description: |
+  luz version 0.3.0 is now on CRAN. luz is a high-level interface for torch.
+author:
+  - name: Daniel Falbel
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+date: 2022-08-23
```

Comment:
can you please also add categories, like so:

categories:
  - Torch
  - Packages/Releases
  - R